src/memray/reporters/assets/temporal_flamegraph.js

```diff
@@ -27,9 +27,8 @@ var parent_index_by_child_index = (function () {
   return ret;
 })();
 
-function packedDataToTree(packedData, rangeStart, rangeEnd) {
-  const { strings, nodes, unique_threads } = packedData;
-
+function generateNodeObjects(packedData) {
+  const { strings, nodes } = packedData;
```

Comment:
Since we only need 2 things, let's just have the caller pass two parameters, instead of forcing them to pass an object that we can unpack here